pkgs/applications/audio/faust/faust2lv2.nix

```diff
@@ -10,6 +10,6 @@ faust.wrapWithBuildEnv {
 
   baseName = "faust2lv2";
 
-  propagatedBuildInputs = [ boost lv2 qt4 which ];
+  propagatedBuildInputs = [ boost lv2_1_16 qt4 which ];
```

Comment:
What is the error here?

Reply:
faust2lv2 is a helper script for the faust compiler.
The package itself compiles just fine, with or without the patch applied, but when you use it to compile a faust program, you get the same error as the other packages:

```
lv2ui.cpp:1946:3: error: invalid conversion from 'void* (*)(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*)' {aka 'void* (*)(const _LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)'} to 'void* (*)(const LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*)' {aka 'void* (*)(const LV2UI_Descriptor*, const char*, const char*, void (*)(void*, unsigned int, unsigned int, unsigned int, const void*), void*, void**, const LV2_Feature* const*)'} [-fpermissive]
make: *** [Makefile:231: KarplusStrongSingerSlaveui.o] Error 1
```

Reply:
What about https://github.com/grame-cncm/faust/commit/7bad68bb6a61cd120f35ab4dfe65fba486733ea1?

Reply:
I missed that, thanks!

Since we where on unstable already for different reasons, I just updated to the latest commit. For good measure, I updated te llvm version that is used.

Reply:
Can you drop `lv2_1_16` then?